onnx/backend/test/case/test_case.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import namedtuple
+
+TestCase = namedtuple('TestCase', [
```

Comment:
Probably add another two fields: atol, rtol for future use?

Also some comments to explain what the purpose of each field is?

Reply:
atol and rtol should be added in this PR https://github.com/onnx/onnx/pull/628